data/src/main/java/com/bottlerocketstudios/mapsdemo/data/implementations/YelpRepositoryImplementation.kt

```diff
@@ -13,8 +14,9 @@ internal class YelpRepositoryImplementation : YelpRepository, KoinComponent {
     // DI
     private val yelpService: YelpService by inject()
 
-    override suspend fun getBusinessesByLatLng(yelpLatLngSearch: YelpLatLngSearch): Result<List<Business>> =
-        yelpService.getBusinessesByLatLng(yelpLatLngSearch.latitude, yelpLatLngSearch.longitude).map { yelpSearch ->
-            yelpSearch.businesses.map { businessDTO -> businessDTO.convertToBusiness() }
+    override suspend fun getBusinessesByLatLng(latLong: LatLong, radius: Int?): Result<List<Business>> =
+        yelpService.getBusinessesByLatLng(latLong.latitude, latLong.longitude, limit = 50, radius = radius).map { yelpSearch: YelpBusinessesDto ->
```

Comment:
@Josh-Gebbeken It would be nice to put a name to 50 to prevent linters from detecting a magic number. Add something like `private const val MAX_RESULTS = 50` and replace 50 with `MAX_RESULTS`

Also, consider setting the default value of limit in the `YelpService.getBusinessesByLatLng` function definition to `MAX_RESULTS` and not passing any value in for limit here. This would entail moving the `MAX_RESULTS` definition to `YelpService`